manifest: sdk-mcuboot: [nrf noup] zephyr/Kconfig: reinstatement of BOOT_ECDSA_NRF_OBERON (#30516)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

* The following non-PSA Crypto implementations were deprecated:

  * :kconfig:option:`CONFIG_BOOT_ECDSA_TINYCRYPT`
  * :kconfig:option:`CONFIG_BOOT_ECDSA_CC310`

>   * :kconfig:option:`CONFIG_BOOT_ED25519_TINYCRYPT`

Reviewer: Ow, this is another leftover from the v3.4.0 release... Those shouldn't be in the current changelog, as they made it to v3.4.0. @FrancescoSer @greg-fer who can fix this?

Reviewer 2: We could remove the whole entry in this same PR (maybe as a separate commit) and add an new entry for the reinstatement of BOOT_ECDSA_NRF_OBERON. Hope this works.

This entry is already mentioned in the 3.4 release notes - https://nrfconnectdocs.nordicsemi.com/ncs/latest/nrf/releases_and_maturity/releases/release-notes-3.4.0.html#mcuboot

Reviewer: agreed

* Added support for the nRF54LC10A SoC.

> * Restored the :kconfig:option:CONFIG_BOOT_ECDSA_NRF_OBERON Kconfig option (for the Crypto implementation support), which was marked as deprecated in the previous release.

Reviewer: ```suggestion
* Restored the :kconfig:option:`CONFIG_BOOT_ECDSA_NRF_OBERON` Kconfig option (for the Crypto implementation support), which was marked as deprecated in the previous release.
```

* Added support for the nRF54LC10A SoC.

> * Reinstated support for the Crypto implementation under :kconfig:option:`CONFIG_BOOT_ECDSA_NRF_OBERON` Kconfig option. This option was deprecated in the previous release.

Reviewer 2: ```suggestion
* Restored support for the Crypto implementation through the :kconfig:option:`CONFIG_BOOT_ECDSA_NRF_OBERON` Kconfig option, which was deprecated in the previous release.
```

Reviewer: I think the suggestion to change to `through the` changes the meaning and implies that it's the only implementation (it's not).
How about this?
```
* Restored support for the :kconfig:option:`CONFIG_BOOT_ECDSA_NRF_OBERON` implementation, which was wrongfully marked as deprecated in the previous release.
```

Reviewer 2: Agree. But could remove the word wrongfully. suggested text?:

`* Restored the :kconfig:option:`CONFIG_BOOT_ECDSA_NRF_OBERON` Kconfig option (for the Crypto implementation support), which was marked as deprecated in the previous release.`
